gradle/libs.versions.toml

```diff
@@ -0,0 +1,26 @@
+[versions]
+gradle = "7.4"
+gradleSha256 = "8cc27038d5dbd815759851ba53e70cf62e481b87494cc97cfd97982ada5ba634"
+kotlin = "1.6.0"
```

Comment:
Is this accessible in build.gradle.kts? That's where we define the Kotlin version (or the kotlin-dev version for snapshots)

Reply:
I've done in https://github.com/Goooler/ktlint/commit/8d719fd85b8e3c4bf2c43574f5bd6ff4112789ee, but something wrong with checksums, it might be a bug of Gradle when using VC in buildSrc with checksums enabled, should we delete `verification-metadata.xml`?

Reply:
Filed a bug to https://github.com/gradle/gradle/issues/20234.

Reply:
> I've done in [Goooler@8d719fd](https://github.com/Goooler/ktlint/commit/8d719fd85b8e3c4bf2c43574f5bd6ff4112789ee), but something wrong with checksums, it might be a bug of Gradle when using VC in buildSrc with checksums enabled, should we delete `verification-metadata.xml`?

Your change [Use VC in buildSrc](https://github.com/Goooler/ktlint/commit/8d719fd85b8e3c4bf2c43574f5bd6ff4112789ee) addresses the problem of the distinct kotlinDev version.  In [this commit](https://github.com/pinterest/ktlint/commit/7487b474a02c97fbfd82fb13dac95484383da196) the version was actually changed and has been merged to master already. Merging this PR would result in reverting the kotlinDev version in the snapshot. If you can mitigate this, I am happy to merge the PR.

Reply:
I've tried again in https://github.com/Goooler/ktlint/pull/19/commits/08fefbe2d99f8923f316bea7b876d7f682da4498, but still failed.
The problem is related to dependency verification, it works if I delete `verification-metadata.xml`.